shotover-proxy/benches/windsock/aws/mod.rs

```diff
@@ -278,28 +298,34 @@ impl Ec2InstanceWithShotover {
             .ssh()
             .push_file_from_bytes(topology.as_bytes(), Path::new("topology.yaml"))
             .await;
+        RunningShotover::new(&self.instance).await
+    }
+}
+
+pub struct RunningShotover {
+    shutdown_tx: tokio::sync::mpsc::UnboundedSender<()>,
+    event_rx: tokio::sync::mpsc::UnboundedReceiver<Event>,
+}
 
+impl RunningShotover {
+    async fn new(instance: &Ec2Instance) -> Self {
         let (shutdown_tx, mut shutdown_rx) = tokio::sync::mpsc::unbounded_channel();
         let (event_tx, mut event_rx) = tokio::sync::mpsc::unbounded_channel();
+        let mut receiver = instance
+                    .ssh()
+                    .shell_stdout_lines(r#"
+        killall -w shotover-bin > /dev/null || true
```

Comment:
I normally use a -9 to make sure things die fast :D

Reply:
IMO its nice to be able to exercise our clean shutdown process here.